core/pom.xml

```diff
@@ -222,6 +222,18 @@
             <version>1.1.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.littleshoot</groupId>
+            <artifactId>littleproxy</artifactId>
+            <version>1.0.0-beta7</version>
```

Comment:
yes, this is exactly the issue I have with this... OTOH one of the next steps is to put the plugin manager into its own own project.. then it would be ok I guess, as it does not pollute the core (the above issue of course does not go away, but at least on a much smaller scale and I do not have any other workaround on my mind)? We could just postpone this PR for that long?